### PR TITLE
[ng] drop EventEmitter from docs and examples

### LINKED ISSUE #566: [ng] Drop EventEmitter from the docs and examples
Cf. https://pub.dartlang.org/packages/angular2/versions/3.0.0-beta%2B2#beta2:
> EventEmitter is now `@Deprecated`: Use Stream and StreamController.

Contributes to #507

Update:
- [ ] docs & examples - #572
- [x] cheatsheet - #569, https://github.com/dart-lang/angular2/pull/359

### PR DESCRIPTION
**Pull request by chalin**
Fixes #566

### CODE REVIEW

File: src/angular/guide/template-syntax.jade [hunk 1]
```diff
@@ -978,14 +978,13 @@ a#event-binding
...
 
-  Directives typically raise custom events with an Angular [EventEmitter](/angular/api/angular2.core/EventEmitter-class.html).
-  The directive creates an `EventEmitter` and exposes it as a property.
-  The directive calls `EventEmitter.emit(payload)` to fire an event, passing in a message payload, which can be anything.
+  Directives typically raise custom events using a [StreamController]({{site.dart_api}}/dart-async/StreamController-class.html).
+  The directive creates a `StreamController` and exposes its underlying `stream` as a property.
+  The directive calls `StreamController.add(payload)` to fire an event, passing in a message payload, which can be anything.
   Parent directives listen for the event by binding to this property and accessing the payload through the `$event` object.
 
   Consider a `HeroDetailComponent` that presents hero information and responds to user actions.
```
Comment (kwalrath) [inline src/angular/guide/template-syntax.jade]: We should add a note about how EventEmitter has been deprecated.

Comment (chalin) [inline src/angular/guide/template-syntax.jade]: My reasoning was that the API docs (and IDEs) will warn that `EventEmitter` is deprecated. So I figured that we could keep the docs simple, and only show/mention the way to do this moving forward (esp. given that there isn't a one-to-one correspondence between `EventEmitter` and how we encode things now).

Do you still want a note added?

Comment (kwalrath) [inline src/angular/guide/template-syntax.jade]: Will it be easy for them to figure out how to replace EventEmitter code? That's what really matters. Not just that it's deprecated, but what to replace it with.

Comment (chalin) [inline src/angular/guide/template-syntax.jade]: IMHO, it is well illustrated in the [Custom events](https://angulardart-org-dev.firebaseapp.com/angular/guide/template-syntax#custom-events) section of the Template Syntax page; in particular the simple code excerpt of **lib/hero_detail_component.dart (deleteRequest)**. Let me know if you agree.

Comment (chalin) [inline src/angular/guide/template-syntax.jade]: Btw, there is an example of the translation in the [API docs of `EventEmitter`](https://github.com/dart-lang/angular2/blob/master/lib/src/facade/async.dart#L3-L19).

Comment (chalin) [inline src/angular/guide/template-syntax.jade]: Cf. https://github.com/dart-lang/angular2/commit/3b711282f841a52479228c93aa1f577e858dbb2a#commitcomment-21907132 for a possible tweak to that API doc.

Comment (kwalrath) [inline src/angular/guide/template-syntax.jade]: I'm fine with no note, now that there's a good example in the EventEmitter docs.

**Pull request merged**